tests/conditional_compilation/tests_utils.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+""" Utility functions for work with json test configuration file.
+"""
+import json
+
+from inspect import getsourcefile
+from pathlib import Path
+
+
+TEST_INFO_NAME = "cc_tests.json"
+
+
+def read_test_info(path: Path = Path(getsourcefile(lambda: 0)).parent / TEST_INFO_NAME):
+    with open(path, 'r') as json_file:
+        cc_tests_ids = json.load(json_file)
+    return cc_tests_ids
+
+
+def write_test_info(path: Path = Path(getsourcefile(lambda: 0)).parent / TEST_INFO_NAME,
+                    data: dict = None):
```

Comment:
I'm not sure `data` should have a default value. Is there a reason for it?

Reply:
`data` still have default value

Reply:
If I remove the default value, I will get an error: non-default parameter follows default parameter

Reply:
why should I remove the default values?